core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -38,9 +41,60 @@
 
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
+    private static volatile DistributionStatisticConfig distributionStatisticConfig =
+            DistributionStatisticConfig.builder()
+                                       .percentiles(PERCENTILES)
+                                       .build();
+
+    /**
+     * Sets {@link DistributionStatisticConfig} to {@link MoreMeters}.
+     * Any {@link Timer} and {@link DistributionSummary} which created by {@link MoreMeters}
+     * using {@link MoreMeters#newTimer(MeterRegistry, String, Iterable)} and
+     * {@link MoreMeters#newDistributionSummary(MeterRegistry, String, Iterable)} will use this config.
+     */
+    public static void setDistributionStatisticConfig(DistributionStatisticConfig config) {
+        requireNonNull(config, "distributionStatisticConfig");
+        distributionStatisticConfig = config;
+    }
+
+    /**
+     * Returns the current {@link DistributionStatisticConfig} of {@link MoreMeters}.
```

Comment:
- How about adding `@see #setDistributionStatisticConfig(..)`?
- `Returns the {@link DistributionStatisticConfig} to use when the factory methods in {@link MoreMeter} create
a {@link Timer} or a {@link DistributionSummary}.`?